users/templates/base.html

```diff
@@ -10,6 +10,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Maybe it would be better to use the newest version of font-awesome ? It offer much more fonts to choose and more other stuffs. I have added it in my PR -> https://github.com/Code-Poets/sheetstorm/pull/508/commits/be25430e3da3bb24fbc5fc39d3e0b4b14ad3a168 in `base.html`